src/simd_basic.h

```diff
@@ -62,6 +62,19 @@ inline VecType CardMinusOne<uint32_t>(uint32_t q)
     return (q == F3) ? F3_MINUS_ONE_U32 : F4_MINUS_ONE_U32;
 }
 
+template <typename T>
+inline VecType GetLowHalf(VecType x, T q)
```

Comment:
No CamelCase for function name (should be `get_low_half`)

Reply:
- [x] rename